www/js/lib/uiUtil.js

```diff
@@ -307,10 +309,16 @@ define(rqDef, function(settingsStore) {
     }
 
     /**
+     * DEV: This function is no longer used in the project and could be removed unless it is of historical interest.
```

Comment:
I think it would be better to remove it if it's unused.
We still have it in git history if needed